server/core/store.go

```diff
@@ -104,19 +109,20 @@ func (s *StoreInfo) Clone(opts ...StoreCreateOption) *StoreInfo {
 // ShallowClone creates a copy of current StoreInfo, but not clone 'meta'.
 func (s *StoreInfo) ShallowClone(opts ...StoreCreateOption) *StoreInfo {
 	store := &StoreInfo{
-		meta:                s.meta,
-		storeStats:          s.storeStats,
-		pauseLeaderTransfer: s.pauseLeaderTransfer,
-		slowStoreEvicted:    s.slowStoreEvicted,
-		leaderCount:         s.leaderCount,
-		regionCount:         s.regionCount,
-		leaderSize:          s.leaderSize,
-		regionSize:          s.regionSize,
-		pendingPeerCount:    s.pendingPeerCount,
-		lastPersistTime:     s.lastPersistTime,
-		leaderWeight:        s.leaderWeight,
-		regionWeight:        s.regionWeight,
-		limiter:             s.limiter,
+		meta:                 s.meta,
+		storeStats:           s.storeStats,
+		pauseLeaderTransfer:  s.pauseLeaderTransfer,
+		slowStoreEvicted:     s.slowStoreEvicted,
+		leaderCount:          s.leaderCount,
+		regionCount:          s.regionCount,
+		leaderSize:           s.leaderSize,
+		regionSize:           s.regionSize,
+		pendingPeerCount:     s.pendingPeerCount,
+		lastPersistTime:      s.lastPersistTime,
+		leaderWeight:         s.leaderWeight,
+		regionWeight:         s.regionWeight,
+		limiter:              s.limiter,
+		operatorExecutorRate: s.operatorExecutorRate,
```

Comment:
Does it mean that it can be different in stores?

Reply:
yes, I think it can be different in future.